docs/configuration/alertmanager-integration/_testing_integration.rst

```diff
@@ -3,9 +3,36 @@ Verify it Works
 
 Send a dummy alert to AlertManager:
 
-.. code-block:: bash
+.. tab-set::
 
-    robusta demo-alert
+    .. tab-item:: Robusta CLI
+
+        If you have the Robusta CLI installed, you can send a test alert using the following command:
+
+        .. code-block:: bash
+
+            robusta demo-alert
+
+    .. tab-item:: Robusta UI
+
+        In the Robusta UI, go to the "Clusters" tab, choose the right cluster and click "Simulate Alert".
+        
+        .. image:: /images/robusta-ui-simulate-alert-1.png
+           :alt: Choose the cluster
+           :width: 600
+           :align: center
+
+        Then 
+
+        1. Check **Send alert with no resource**.
+        2. Provide a name for the alert in the **Alert name (identifier)** field (e.g., "Testing Prod AlertManager").
+        3. Select **Alert Manager** under the "Send alert to" section.
+        4. Click the **Simulate Alert** button to send the test alert.
+
+        .. image:: /images/robusta-ui-simulate-alert-2.png
+           :alt: Send Test Alert
+           :width: 600
+           :align: center
 
 If everything is setup properly, this alert will reach Robusta. It will show up in the Robusta UI, Slack, and other configured sinks.
```

Comment:
I would add a note, that it might take a few minutes for the alert to arrive, because of alertmanager `group_wait` and `group_interval` [see here](https://prometheus.io/docs/alerting/latest/configuration/#:~:text=How%20long%20to%20wait%20before%20sending%20a%20notification%20about%20new%20alerts%20that%0A%23%20are%20added%20to%20a%20group%20of%20alerts%20for%20which%20an%20initial%20notification%20has%0A%23%20already%20been%20sent)